knowledge-base/pass-sqldatasource-columns-as-datasource-for-nested-data-items.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to Use Part of the Columns from a Parent SqlDataSource as DataSource for Nested Data Items
+description: When a parent data item is bound to a SqlDataSource you may still use part of the returned columns as DataSource for child data items
+type: how-to
+page_title: How to Pass SqlDataSource Columns as DataSource for Nested Data Items
+slug: pass-sqldatasource-columns-as-datasource-for-nested-data-items
+position: 
+tags: 
+ticketid: 1454415
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+When you need to pass data from a [SqlDataSource](../sqldatasource) bound to a data item to its nested [data item](../data-items), you need to pass all of the data. This is due to the fact that SqlDataSource can return only a single DataTable. Even if the query returns a DataSet, only the first of the tables will be used - [Selecting Data with the SqlDataSource component](../sql-data-source-selecting-data). In some cases, this may significantly deteriorate the performance.  
```

Comment:
When I read "nested data items", I usually think of real "nested" items like graph in a table. I would use the "master-details" term again just to explain for which scenarios this KB is suitable. Most users are familiar with "M-D", but do not clearly understand what "nested items" mean (or at least I think so).